test/e2e/arktos/multi_tenancy/run-e2e-test.sh

```diff
@@ -23,7 +23,10 @@ script_root=$(dirname "${BASH_SOURCE}")
 repo_root=$(cd $(dirname $0)/../../../.. ; pwd)
 
 #put the test suite file names below, one line one suite. The test suites will be run in the order defined.
-test_suite_files="tenant_init_delete_test.yaml"
```

Comment:
it seems test code is sort-of mixed with test data files. Could they be further separated, so that when new test cases are added, we don't have to touch this test code file at all?

Reply:
Actually, the code and the test suite files are separate. The code after under "cmd" subfolder, while the test suite files are under subfolder "test_suite".